lib/l10n/app_en.arb

```diff
@@ -2,6 +2,7 @@
   "email": "Email",
   "login": "Log in",
   "password": "Password",
+  "auth_error": "Email or password format is not valid. Please try again!",
```

Comment:
Can we have a better name for this string? For example, `login_validation_error`?